FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,46 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] httpResult in
+			switch httpResult {
+			case .failure:
+				completion(.failure(RemoteFeedLoader.Error.connectivity))
```

Comment:
`RemoteFeedLoader` can be omitted since this portion of code is in the same scope, making the code shorter:
```suggestion
				completion(.failure(Error.connectivity))
```